lexers/v/testdata/vb_if.vb

```diff
@@ -0,0 +1 @@
+#If DEBUG Then
```

Comment:
You always check files, where the pattern is on the first line. Thus the tests do not catch the limitation of the regex usage, which I remarked below. In matlab testdata files, I also added blank lines to the testfiles: https://github.com/wakatime/chroma/blob/master/lexers/m/testdata/matlab_systemcmd.m